security/email-templates.md

```diff
@@ -21,7 +21,7 @@ defect(s). The highest rated security defect is considered $SEVERITY severity.
 No further details or patches will be made available in advance of the release.
 
 Thanks,
-$PERSON (on behalf of the Envoy maintainers)
+$PERSON (on behalf of the Envoy security team)
```

Comment:
+1 Makes sense for members of PST who aren't in maintainers. 
(Presuming it's clear that the announcement is endorsed and coming from maintainers since they're a subset of sec team. Otherwise maybe `Envoy security team and maintainers`)

Reply:
+1 for security team + maintainers but don't feel strongly about it.

Reply:
done